packages/astro/src/core/build/internal.ts

```diff
@@ -114,6 +116,8 @@ export function createBuildInternals(): BuildInternals {
 		discoveredScripts: new Set(),
 		staticFiles: new Set(),
 		componentMetadata: new Map(),
+		ssrServerlessEntryChunks: new Map(),
```

Comment:
nit: I think it would be nice to remove the term "serverless" around the codebase since it's not specifically for serverless environments.

I'm not sure what others think about this though, but it's a bit confusing for me if we keep the term here.